account/tests.py

```diff
@@ -338,3 +338,127 @@ def test_password_is_only_number(self):
+    """
+    ログイン機能に対するテスト
+    """
+
+    def setUp(self):
+        account = Account.objects.create_user(
+            email='sample@example.com', 
+            username='sample'
+        )
+        account.set_password('instance1')
+        account.save()
+        self.path = reverse('account:login')
+        self.client = Client()
+
+    def test_correct_login(self):
+        """
+        正しくログインした場合
+        """
+
+        saved_accounts = Account.objects.all()
+        self.assertEqual(saved_accounts.count(), 1)
+
+        login = self.client.login(username='sample', password='instance1')
+
+        self.assertTrue(login)
+
+        response = self.client.post(
+            path=self.path,
+            data={
+                'username': 'sample',
+                'password': 'instance1'
+            }
+        )
+        self.assertEqual(response.status_code, 200)
+        self.assertTrue(response.context['user'], 'sample')
+
+    def test_incorrect_password(self):
+        """
+        間違ったパスワードを入力した場合
+        """
+
+        saved_accounts = Account.objects.all()
+        self.assertEqual(saved_accounts.count(), 1)
+
+        login = self.client.login(username='sample', password='instance2')
+        self.assertFalse(login)
+
+        response = self.client.post(
+            path=self.path,
+            data={
+                'username': 'sample',
+                'password': 'instance2'
+            }
+        )
+        self.assertEqual(response.status_code, 200)
+
+        form = LoginForm(
+            data={
+                'username': 'sample',
+                'password': 'instance2'
+            }
+        )
+        self.assertEqual(
+            form.errors['__all__'][0], 
+            '正しいユーザー名とパスワードを入力してください。どちらのフィールドも大文字と小文字は区別されます。'
+        )
+
+    def test_not_registered_account(self):  
+        """
+        登録していないアカウントの情報を入力した場合
+        """
+        
+        saved_accounts = Account.objects.all()
+        self.assertEqual(saved_accounts.count(), 1)
+
+        login = self.client.login(username='example', password='instance10')
+        self.assertFalse(login)
+
+        response = self.client.post(
+            path=self.path,
+            data={
+                'username': 'example',
+                'password': 'instance10'
+            }
+        )
+        self.assertEqual(response.status_code, 200)
+
+        form = LoginForm(
+            data={
+                'username': 'example',
+                'password': 'instance10'
+            }
+        )
+        self.assertEqual(
+            form.errors['__all__'][0], 
+            '正しいユーザー名とパスワードを入力してください。どちらのフィールドも大文字と小文字は区別されます。'
+        )
+
+
+class LogoutTest(TestCase):
+    """
+    ログアウト機能に対するテスト
+    """
+
+    def setUp(self):
+        account = Account.objects.create_user(
+            email='sample@example.com', 
+            username='sample'
+        )
+        account.set_password('instance1')
+        account.save()
```

Comment:
同様に、こちらの方がシンプルで良いと思います。
```suggestion
       Account.objects.create_user(
            email='sample@example.com', 
            username='sample',
            password='instance1'
        )
```